server/src/main/java/org/elasticsearch/action/search/AbstractSearchAsyncAction.java

```diff
@@ -543,8 +543,10 @@ protected final SearchResponse buildSearchResponse(InternalSearchResponse intern
             skippedOps.get(), buildTookInMillis(), failures, clusters, searchContextId);
     }
 
-    boolean includeSearchContextInResponse() {
-        return request.pointInTimeBuilder() != null;
+    boolean buildPointInTimeFromSearchResults() {
+        // TODO: Until we implement the retry mechanism for point in times (i.e., replace an unavailable shard with an equivalent copy),
```

Comment:
I noticed this TODO while working on an unrelated refactor. Do we need to address the TODO now that we retry point-in-times (#66713)?